lib/tasks/hathi_etas.rake

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+HATHI_DATA_HOME = '/data/hathitrust_data'
+
+namespace :hathitrust do
+  desc 'Process overlap file for emergency access to restricted HathiTrust material'
+  task :process_hathi_etas, [:hathi_full, :overlap_psu] do |_task, args|
+    Dir.chdir(HATHI_DATA_HOME.to_s) do
+      Rake::Task['hathitrust:pare_hathi_full'].invoke(args[:hathi_full])
+      Rake::Task['hathitrust:extract_overlap_oclc'].invoke(args[:overlap_psu])
+      Rake::Task['hathitrust:filter_overlap'].invoke
+      Rake::Task['hathitrust:extract_uniq_oclc'].invoke
+    end
+  end
+
+  desc 'Pare the monthly file down to just the needed data: Hathi Trust id and OCLC number'
+  task :pare_hathi_full, [:hathi_full] do |_task, args|
+    print `gunzip -c #{args[:hathi_full]} | \
```

Comment:
should we just make it wget/curl the full data set? I think it has a specific pattern for file naming

Reply:
> wonders if these tools are installable via `yum` and if so we need to add them (preferably through our deploy project)

that's what I was kinda hoping so as well that we can install them with our deploy process

Reply:
yes, I think so, I will give it a try

Reply:
added this